config/database.yml

```diff
@@ -29,15 +29,15 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  #username: govuk_rails_boilerplate
+  username: postgres
```

Comment:
Could we not change these defaults and instead set `DATABASE_URL` environment variable in the docker-compose file? 🙏